SbtReactiveAppPlugin does not work with SBT 1.1.1

Project for reference: https://github.com/mjcramer/hello-reactive-tooling.git

When the sbt version is updated to 1.1.1 (current stable release) it appears that the plugin breaks and cannot load:

`[info] Loading settings from build.sbt ...
[error] java.lang.ExceptionInInitializerError
[error] 	at com.lightbend.rp.sbtreactiveapp.SbtReactiveAppLagomScalaPlugin$.projectSettings(SbtReactiveAppPlugin.scala:120)
[error] 	at com.lightbend.rp.sbtreactiveapp.SbtReactiveAppLagomScalaPlugin$.projectSettings(SbtReactiveAppPlugin.scala:110)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$3(Load.scala:1025)
[error] 	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
[error] 	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:352)
[error] 	at sbt.internal.Load$.autoPluginSettings$1(Load.scala:1025)
[error] 	at sbt.internal.Load$.expandSettings$1(Load.scala:1042)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$10(Load.scala:1045)
[error] 	at scala.collection.IndexedSeqOptimized.foldLeft(IndexedSeqOptimized.scala:56)
[error] 	at scala.collection.IndexedSeqOptimized.foldLeft$(IndexedSeqOptimized.scala:64)
[error] 	at scala.collection.mutable.WrappedArray.foldLeft(WrappedArray.scala:38)
[error] 	at scala.collection.TraversableOnce.$div$colon(TraversableOnce.scala:151)
[error] 	at scala.collection.TraversableOnce.$div$colon$(TraversableOnce.scala:151)
[error] 	at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:104)
[error] 	at sbt.internal.Load$.expandSettings$1(Load.scala:1044)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$1(Load.scala:1048)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.resolveProject(Load.scala:1014)
[error] 	at sbt.internal.Load$.finalizeProject$1(Load.scala:864)
[error] 	at sbt.internal.Load$.discoverAndLoad$1(Load.scala:883)
[error] 	at sbt.internal.Load$.loadTransitive(Load.scala:892)
[error] 	at sbt.internal.Load$.loadProjects$1(Load.scala:714)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$10(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:680)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:153)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:136)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:136)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:136)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:129)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:107)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:101)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:57)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:42)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:34)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:109)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:117)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:18)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:17)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] Caused by: java.lang.IllegalArgumentException: requirement failed: id must be capitalized: apiTools
[error] 	at scala.Predef$.require(Predef.scala:277)
[error] 	at sbt.librarymanagement.Configuration.<init>(Configuration.scala:19)
[error] 	at sbt.librarymanagement.Configuration$.of(Configuration.scala:72)
[error] 	at com.lightbend.rp.sbtreactiveapp.LagomApp.$init$(App.scala:51)
[error] 	at com.lightbend.rp.sbtreactiveapp.LagomScalaApp$.<init>(App.scala:126)
[error] 	at com.lightbend.rp.sbtreactiveapp.LagomScalaApp$.<clinit>(App.scala)
[error] 	at com.lightbend.rp.sbtreactiveapp.SbtReactiveAppLagomScalaPlugin$.projectSettings(SbtReactiveAppPlugin.scala:120)
[error] 	at com.lightbend.rp.sbtreactiveapp.SbtReactiveAppLagomScalaPlugin$.projectSettings(SbtReactiveAppPlugin.scala:110)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$3(Load.scala:1025)
[error] 	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
[error] 	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:352)
[error] 	at sbt.internal.Load$.autoPluginSettings$1(Load.scala:1025)
[error] 	at sbt.internal.Load$.expandSettings$1(Load.scala:1042)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$10(Load.scala:1045)
[error] 	at scala.collection.IndexedSeqOptimized.foldLeft(IndexedSeqOptimized.scala:56)
[error] 	at scala.collection.IndexedSeqOptimized.foldLeft$(IndexedSeqOptimized.scala:64)
[error] 	at scala.collection.mutable.WrappedArray.foldLeft(WrappedArray.scala:38)
[error] 	at scala.collection.TraversableOnce.$div$colon(TraversableOnce.scala:151)
[error] 	at scala.collection.TraversableOnce.$div$colon$(TraversableOnce.scala:151)
[error] 	at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:104)
[error] 	at sbt.internal.Load$.expandSettings$1(Load.scala:1044)
[error] 	at sbt.internal.Load$.$anonfun$resolveProject$1(Load.scala:1048)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.resolveProject(Load.scala:1014)
[error] 	at sbt.internal.Load$.finalizeProject$1(Load.scala:864)
[error] 	at sbt.internal.Load$.discoverAndLoad$1(Load.scala:883)
[error] 	at sbt.internal.Load$.loadTransitive(Load.scala:892)
[error] 	at sbt.internal.Load$.loadProjects$1(Load.scala:714)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$10(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:680)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:153)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:136)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:136)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:136)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:129)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:107)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:101)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:57)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:42)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:34)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:109)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:117)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:18)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:17)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.ExceptionInInitializerError
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?`

--- a/src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala
+++ b/src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala
@@ -48,7 +48,7 @@ sealed trait App extends SbtReactiveAppKeys {
 }
 
 sealed trait LagomApp extends App {
-  val apiTools = config("api-tools").hide
+  val ApiTools = config("api-tools").hide
 
   def projectSettings: Seq[Setting[_]] =
     Vector(
@@ -65,12 +65,12 @@ sealed trait LagomApp extends App {
 
       enableServiceDiscovery := true,
 
-      ivyConfigurations += apiTools,
+      ivyConfigurations += ApiTools,
 
-      managedClasspath in apiTools :=
-        Classpaths.managedJars(apiTools, (classpathTypes in apiTools).value, update.value),
+      managedClasspath in ApiTools :=
+        Classpaths.managedJars(ApiTools, (classpathTypes in ApiTools).value, update.value),
 
-      libraryDependencies ++= magic.Lagom.component("api-tools").toVector.map(_ % apiTools),
+      libraryDependencies ++= magic.Lagom.component("api-tools").toVector.map(_ % ApiTools),
 
       lagomRawEndpoints := {
         val ingressPorts = httpIngressPorts.value
@@ -80,7 +80,7 @@ sealed trait LagomApp extends App {
 
         val magicEndpoints =
           magic.Lagom.endpoints(
-            ((managedClasspath in apiTools).value ++ (fullClasspath in Compile).value).toVector,
+            ((managedClasspath in ApiTools).value ++ (fullClasspath in Compile).value).toVector,
             scalaInstance.value.loader,
             ingressPorts.toVector,
             ingressHosts.toVector,

